app/controllers/course/courses_controller.rb

```diff
@@ -8,7 +8,7 @@ def index # :nodoc:
 
   def show # :nodoc:
     @registration = Course::Registration.new
-    @activity_feeds = recent_activity_feeds.limit(20).includes(:activity)
+    @activity_feeds = recent_activity_feeds.limit(20).includes(activity: [:object, :actor])
```

Comment:
@lowjoel any thoughts on how to do it as a scope? I'm not sure how to do it because of polymorphism.